prisma/schema.prisma

```diff
@@ -10,3 +10,11 @@ datasource db {
   url               = env("DATABASE_URL")
   shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
 }
+
+model Pokemon {
+    id          Int       @id @default(autoincrement())
+    name        String
+    art         String
+    type1       String
+    type2       String?
```

Comment:
I think the original generation of pokemon only ever have 1 or 2 types so this is okay, but it's worth thinking about how this would look if you wanted to expand your app to include newer generations in future.

Another approach you could have taken would be to add a `Type` model and use relationships to connect pokemon to types

Reply:
having a Type model was our initial idea but we realised that this was much more simple for what we were trying to accomplish. We also discussed having a many-to-many relationship vs a one-to-many with the different Type models and the different ways to find/filter the database.